atlasdb-tests-shared/src/main/java/com/palantir/timelock/paxos/InMemoryTimelockServices.java

```diff
@@ -126,6 +147,23 @@ protected void before() throws IOException {
                 .until(() -> delegate.getTimestampService().getFreshTimestamp() > 0);
     }
 
+    private void createHelperServices(MetricsManager metricsManager) {
+        helperServices = TimeLockHelperServices.create(
+                client,
+                metricsManager,
+                ImmutableSet.of(),
+                delegate.getTimelockService(),
+                LockWatchCachingConfig.builder().build(),
+                Optional::empty);
+
+        RedirectRetryTargeter redirectRetryTargeter = timeLockAgent.redirectRetryTargeter();
+        ConjureTimelockService cts =
+                ConjureTimelockResource.jersey(redirectRetryTargeter, _unused -> delegate.getTimelockService());
+        NamespacedConjureTimelockService ncts = new NamespacedConjureTimelockServiceImpl(cts, client);
+        LeaderTimeGetter ltg = new LegacyLeaderTimeGetter(ncts);
+        lockLeaseService = LockLeaseService.create(ncts, ltg);
```

Comment:
I appreciate that Atlas doesn't have a good record of variable names, but please can we bias towards using `timelockService` or `conjureTimelockService` over `cts`, and likewise?